### ISSUE TITLE
Export a simple list of marked files with this simple format: "duplicate/path,reference/path"

### ISSUE THREAD

**Issue by illwieckz** [hsoft/dupeguru#320]
Hi, this software is very good to find duplicates and I like so much the way it allows me to compared duplicates and references to check if it's really a duplicates or not. :+1: 

Thanks a lot for all this good work.

But when I marked all my duplicates (or unmarked all the false positives) I can't do nothing useful with the file generate the duplicate list, it seems there is no way to export a simple “`duplicate/path`,`reference/path`" readable by third-party tools, the already existing exports options does not do this simple task. Even the _“Export to CSV"_ option does not do that.

I'm working on assets that references some pictures, so, when I deduplicate pictures, I need to update the references. For example, imagine you have some HTML files that references some pictures, when you delete the duplicated files, you need to substitute the deleted files path with the reference path in each <img> tag, so you need a list with duplicate path and reference path so you can automatize a substitution.

Is it possible to add this simple export format? It just need to export a csv file with this format: "`duplicate/path`,`reference/path`".

There is in dupeguru an option to run a personalized command, so I can already run a personalized command that just do `echo "%d,%r" >> /tmp/duplicate.csv`, but you know, you can only run a personalized on only one file at a time, not for the complete list of marked files, so if I deduplicate thousand of files, I need to do Ctrl-Alt-I thousands of time to generate the whole csv, which is very painful. :confused:

**Comment by hsoft** [comment]
It's a good opportunity for a first PR, which is why I flag this issue as mentored.

I remember having considered making the custom command work on all selected dupes rather than just one, but I can't find the corresponding issue. It would be a good idea as well.

**Comment by illwieckz** [comment]
Yes, having a way to run the customized command for all the selected dupes or all the marked dupes will solves my problem, even it there is no explicit "simple csv export" support added.

**Comment by hsoft** [comment]
I remember now about a caveat: Some custom commands involve spawning some kind of window in a GUI. When that's the case, it's possible to mistakenly crowd your desktop with hundreds of windows, so I thought it would be a good idea to ask for confirmation when selection is more than 10 items.

**Comment by illwieckz** [comment]
Oh yes :scream: . Running some GUI tool this way is a thing I will never do myself, but other people can…
